apps/site/assets/ts/schedule/components/schedule-finder/daily-schedule/ScheduleTable.tsx

```diff
@@ -74,9 +74,7 @@ const ScheduleTable = ({
       <table className="schedule-table">
         <thead className="schedule-table__header">
           <tr>
-            {anySchoolTrips && (
-              <th scope="col" className="schedule-table__cell" />
-            )}
+            {anySchoolTrips && <td className="schedule-table__cell" />}
```

Comment:
Is this right? should we be mixing `td` and `th`?

Reply:
Actually, yes! :D That's only because it's an empty cell we use for spacing purposes -- [table headers should never be empty, but empty table cells, even in the header, are ok](https://www.w3.org/TR/WCAG20-TECHS/H51.html).

Reply:
learn something new every day!